galley/pkg/crd/validation/validation.go

```diff
@@ -148,10 +148,10 @@ func isDNS1123Label(value string) bool {
 
 // validatePort checks that the network port is in range
 func validatePort(port int) error {
-	if 1 <= port && port <= 65535 {
+	if 1024 <= port && port <= 65535 {
 		return nil
 	}
-	return fmt.Errorf("port number %d must be in the range 1..65535", port)
+	return fmt.Errorf("port number %d must be in the range 1024..65535", port)
```

Comment:
why can't it be below 1024? The user can still run as root if they want and use port 443 right?

Reply:
I think we should run the pod as non-root by default. the root user is not recommended way. if we keep the validation for 1..65535, it throws the exception for nonroot user when the port number is less than 1024. I think we should validate it.

Reply:
You are now saying that a user CANNOT run as root.

Also this has upgrade implications, users now cannot upgrade from 1.2 to 1.3 without changing their deployments